molSimplify/Scripts/rmsd.py

```diff
@@ -472,6 +477,77 @@ def rigorous_rmsd(mol1, mol2, rotation: str = "kabsch",
                                       rotation=rotation, reorder=reorder)
     return result_rmsd
 
+def align_rmsd(mol1, mol2, rotation: str = "kabsch",
+               reorder: str = "hungarian") -> float:
+    """
+    Computes the RMSD between 2 mol objects after:
+    - translating them both such that the center of mass is at the origin
+    - projecting the coordinates onto the principal axes
+    - reordering x, y, z such that Ixx < Iyy < Izz
+
+    Parameters
+    ----------
+        mol1 : mol3D
+            mol3D instance of initial molecule.
+        mol2 : np.mol3D
```

Comment:
'mol2' is potentially a confusing name since the code uses mol2 strings widely; consider using 'mol_a' and 'mol_b' or another naming convention